core/arch/arm/plat-k3/platform_config.h

```diff
@@ -97,8 +99,14 @@
 #define SA2UL_REG_SIZE		0x1000
 
 /* RNG */
+#if defined(PLATFORM_FLAVOR_am62lx)
+#define RNG_BASE		(EIP76D_TRNG_BASE)
```

Comment:
I think EIP76D_TRNG_BASE is redundant and can be remove itself, I don't see any other usage of it.

Reply:
Done.